procfile

```diff
@@ -0,0 +1 @@
+web: npm run build
```

Comment:
@izzett222 Eslint

Reply:
when I add a newline , its give it still show the error

Reply:
@AbdoulNuru I fixed the problem